src/commcare_cloud/ansible/roles/nginx/vars/cchq_ssl.yml

```diff
@@ -75,6 +75,14 @@ nginx_sites:
       proxy_buffers: 8 64k
       proxy_buffer_size: 64k
       client_body_buffer_size: 512k
+    - name: '~ /hq/admin/phone/restore/'
```

Comment:
Nit: I see value in both the way it was before and the way it is now. This makes the paths easier to parse visually, but then makes it look like the related configuration variables (`balancer`, `proxy_redirect`, etc.) may be different when in fact they are (and are intended to stay) the same. I think I would probably go with just having a single

```
    - name: '~ (/a/[^/]+/(receiver|phone)/|/hq/admin/phone/restore/)'
```
above. That way the two different patterns are clearly differentiated (both for anyone familiar with regex willing to look at it for a couple seconds, as well as for some likely text searches).

As for the `/a/{domain}/phone/admin_restore/` endpoint (is that the right URL?), I think your reasoning (which seems right to me) is that the existing pattern already covers that. Just wanted to call that out explicitly in case you disagree.

Reply:
> As for the /a/{domain}/phone/admin_restore/ endpoint (is that the right URL?), I think your reasoning (which seems right to me) is that the existing pattern already covers that. Just wanted to call that out explicitly in case you disagree.

Yeah that is the reasoning. Thanks for calling that out explicitly.

And yeah confirmed that the `admin_restore` URL is `/a/{domain}/phone/admin_restore/`